src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -42,7 +42,20 @@ object WorkflowActor {
 
   val DatabaseTimeout = 5 seconds
 
-  type ExecutionStore = Map[Call, ExecutionStatus.Value]
+  type ExecutionStore = Map[ExecutionStoreKey, ExecutionStatus]
+  type ExecutionStoreEntry = (ExecutionStoreKey, ExecutionStatus)
+
```

Comment:
Yes, we'll eventually need _something_ like it. I suspect `populate` will merge with the currently in use `createStore`. I had the function at one point only in the test code, but wanted to leave it closer to the main code for visibility of updating `createStore`.

Will update the comments in both saying as much.